src/main/java/frc/robot/commands/shooter/ShootSpeaker.java

```diff
@@ -28,7 +30,11 @@ public void initialize() {
 
     @Override 
     public boolean isFinished() {
-        return (timer.hasElapsed(Constants.SHOOTER_TIME_AFTER_TRIGGER)) && (activated);
+        if (timer.hasElapsed(Constants.SHOOTER_TIME_AFTER_TRIGGER) && (activated)) {
```

Comment:
This is an interesting command because its only end condition is time. For that reason I don't know that it's that useful to log this. I guess for now leave it and we'll see how much noise it creates.

Reply:
I have made these changes.